server/observable.go

```diff
@@ -699,7 +699,7 @@ func (o *Observable) sampleAck(sseq, dseq, dcount uint64) {
 
 	now := time.Now()
 	e := &ObservableAckSampleEvent{
-		Schema:     "io.nats.jetstream.event.v1.obs_ack",
+		Schema:     "io.nats.jetstream.sample.v1.obs_ack",
```

Comment:
Are schemas that reverse java style notation?

Reply:
I do not believe there is a agreed on convention, many projects do it this way, many don't, you just have to pick a thing. K8s for example does it using 3 different keys where one key is a composite of others :)